rails-task-list/app/views/tasks/index.html.erb

```diff
@@ -1,23 +1,48 @@
 <h1>Tasks</h1>
 
-<div id="button_box">
 
+<a href="/tasks/new">
+<table id="add_task_button">
+  <tr>
+    <td>
+      <i class="fa fa-space-shuttle fa-4x"></i>
+    </td>
+    <td id="add_text">
+      Add a new task
+    </td>
+    <td>
+      <i class="fa fa-space-shuttle fa-4x fa-rotate-180 fa-flip-horizontal"></i>
+    </td>
+  </tr>
+</table>
+</a>
 
+<div id="button_box">
 
-  <form class="add_button" method="get" action="/tasks/new">
-    <i class="fa fa-space-shuttle fa-4x"></i>
-    <input type="submit" value="Add a task">
-    <i class="fa fa-space-shuttle fa-4x fa-rotate-180 fa-flip-horizontal"></i>
-  </form>
+  <div id="background_mod">
 
-</div>
 
   <% @tasks.each do |task| %>
-
-    <a class="list_box" href = "/tasks/<%= task.id %>"> <%= task.name %>
-
-          <!-- <%= link_to task.name, "/tasks/#{task.id}"  %> -->
-          <%= button_to "Delete",  "/tasks/#{task.id}", method: :delete, data: { confirm: "Are you sure you want to delete this task?"}  %>
-
-    </a>
-  <% end %>
+  <a id="task_link" href="/tasks/<%= task.id %>"
+    <% if task.completed_date != nil %>
+      class = "completed_task"
+    <% end %>
+    >
+    <%= task.name %></a>
+
+  </div>
```

Comment:
This `div` tag should probably be moved after the `<% end %>` on line 48. Because the two divs `button_box` and `background_mod` are opened before the loop on line 25, they should not be closed within the loop.